### DISCUSSION TITLE
New to Orca

### DISCUSSION THREAD

**Discussion by danm46** [OrcaSlicer/OrcaSlicer#3372] [Q&A]
I just downloaded Orca and when comparing print times from Prusa to Orca, the print time is doubling when slicing with Orca.
As far as I can see the settings are all fairly equal, just wondering if there is something major im missing.
I am a fairly new printer running a Elegoo neptune 4

![slicer](https://github.com/SoftFever/OrcaSlicer/assets/155256290/8b24ccf5-93ad-4736-b859-f734fdf2f569)

**Comment by stuomas** [comment]
Well other has supports enabled that takes like 25% of total time, and probably the rest is explained by other settings
